### PR TITLE
Google Assistant SDK: Document audio response playback

### PR DESCRIPTION
**Pull request by tronikos**
## Proposed change

Document support for audio response playback in Google Assistant SDK

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85989
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by tronikos** [comment]
@frenck could you please merge this in rc too? It had a merge conflict earlier.

File: source/_integrations/google_assistant_sdk.markdown [hunk 1]
```diff
@@ -109,15 +114,23 @@ You can use the service `google_assistant_sdk.send_text_command` to send command
...
 | `command`              | no       | Command(s) to send to Google Assistant. | turn off kitchen TV |
+| `media_player`         | yes      | Name(s) of media player entities to play response on | media_player.living_room_speaker |
 
-Example:
+Examples:
 
 ```yaml
 service: google_assistant_sdk.send_text_command
 data:
   command: turn off kitchen TV
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]:
```suggestion
  command: "turn off kitchen TV"
```

File: source/_integrations/google_assistant_sdk.markdown [hunk 2]
```diff
@@ -109,15 +114,23 @@ You can use the service `google_assistant_sdk.send_text_command` to send command
...
 ```
 
+```yaml
+# Say a joke on the living room speaker
+service: google_assistant_sdk.send_text_command
+data:
+  command: tell me a joke
```
Comment (frenck) [inline source/_integrations/google_assistant_sdk.markdown]:
```suggestion
  command: "tell me a joke"
```

**Review by frenck — APPROVED** [review]
Thanks, @tronikos 👍 

../Frenck

**Pull request merged**